superset/connectors/druid/models.py

```diff
@@ -31,20 +31,26 @@
 from flask_appbuilder.models.decorators import renders
 from flask_babel import lazy_gettext as _
 import pandas
-from pydruid.client import PyDruid
-from pydruid.utils.aggregators import count
-from pydruid.utils.dimensions import MapLookupExtraction, RegexExtraction
-from pydruid.utils.filters import Dimension, Filter
-from pydruid.utils.having import Aggregation
-from pydruid.utils.postaggregator import (
-    Const,
-    Field,
-    HyperUniqueCardinality,
-    Postaggregator,
-    Quantile,
-    Quantiles,
-)
-import requests
+try:
+    from pydruid.client import PyDruid
+    from pydruid.utils.aggregators import count
+    from pydruid.utils.dimensions import MapLookupExtraction, RegexExtraction
+    from pydruid.utils.filters import Dimension, Filter
+    from pydruid.utils.having import Aggregation
+    from pydruid.utils.postaggregator import (
+        Const,
+        Field,
+        HyperUniqueCardinality,
+        Postaggregator,
+        Quantile,
+        Quantiles,
+    )
+    import requests
+except ImportError:
```

Comment:
It would probably be good to use either `ImportError` or `ModuleNotFoundError` everywhere? As `ModuleNotFoundError` is a child of `ImportError` and thus more precise, I would opt for that.

Reply:
I agree with being uniform but I'm afraid that using `ModuleNotFoundError` doesn't work everywhere, e.g. when importing `Having` if I try to catch `ModuleNotFoundError` it crashes with `ImportError`. I'd try with `ImportError` everywhere if you agree.

Reply:
Ok, then it probably makes sense to keep it as it is.